wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/PocketCastsWearApplication.kt

```diff
@@ -33,10 +40,23 @@ class PocketCastsWearApplication : Application() {
     }
 
     private fun setupApp() {
+        runBlocking {
+            withContext(Dispatchers.Default) {
+                playbackManager.upNextQueue.setup()
+            }
+        }
         userManager.beginMonitoringAccountManager(playbackManager)
     }
 
     private fun setupAnalytics() {
         AnalyticsTracker.init(settings)
     }
+
+    override fun getWorkManagerConfiguration(): Configuration {
+        return Configuration.Builder()
+            .setWorkerFactory(workerFactory)
+            .setExecutor(Executors.newFixedThreadPool(3))
+            .setJobSchedulerJobIdRange(1000, 20000)
```

Comment:
Just curious, did we have to set these values because otherwise the defaults caused problems?

Reply:
I'm not sure about the history around it. To be safe, I kept the values same as were set in mobile and automotive apps. I think it is a decent range that works for us.